submissions.html

```diff
@@ -0,0 +1,149 @@
+     </ul>
+
+     All manuscripts will be reviewed for originality, academic standards, appropriateness of topic, readability (language), and technical feasibility. The audio/video file will be assessed for its artistic/conceptual quality and relevance to the manuscript and the topic.<br><br> 
+     Please note that currently, we are able to provide only minimal technical equipment for the live performances. We will provide a stereo sound system, a projector, and potentially internet connection. Special situations can be discussed, but you should consider bringing your own equipment and musical instruments, whenever applicable. Please make sure to include this information in your tech rider.<br><br>
+
+</ol>
+
+
+<p class='font-medium'>Paper Submission</p>
+<!-- #E40066 color for when submissions are closed. -->
+<p>The paper submission for ISQCMC '25 is currently <a href="mailto:quantummusicsubmission@gmail.com"><span style='color: #03CEA4;'>open</span></a>. Please refer to the detailed instructions below to ensure a smooth peer-review and avoid unnecessary rejections of your submission.</p>
+
+
+The deadline for submission is <strong>May 09, 2025</strong>. <br>
+
+<p>Paper Submission Instructions</p>
+Please make sure the paper is formatted according to the <a href="https://www.overleaf.com/latex/templates/isqcmc-2025-template/qyfngrjtyvjv"><u>ISQCMC 2025 Template</u></a>, is <em> anonymized</em> (see the Anonymization section below),and does not exceed 5 pages.<br><br>
+
+All submissions are to be sent to <a href="mailto:quantummusicsubmission@gmail.com">quantummusicsubmission@gmail.com</a>, and <strong> should contain the following information and files:</strong>
+
+<ul>
+  <li>Full names of all authors, and their respective affiliations (if applicable)</li>
+  <li>Clear indication of the corresponding author(s) and their contact email(s)</li>
+  <li>The <em>anonymized version</em> of your paper, in PDF format</li>
+  <li>Any additional supporting information or appendices</li>
+</ul>
+  
+
+
+
+<p> Anonymization </p>
+Your paper should be anonymized for the double-blind peer-review process. This means that you should hide or censor any information that could identify the authors, such as names, affiliations, or acknowledgments. Any other information that may reveal your identity should be ideally avoided. In case you are using a substantial amount of self-referential citations, you may also consider censoring them while adjusting the writing to imply so (e.g. "In our previous work [censored citation]"). <br>
+
+<p><em>LaTeX Template Anonymization usage:</em></p> The <a href="https://www.overleaf.com/latex/templates/isqcmc-2025-template/qyfngrjtyvjv"><u>ISQCMC 2025 LaTeX Template</u></a> has a feature that submitters can use to anonymize their papers. Its usage is explained below. <br><br>
+
+Check <strong>line 20</strong> of the template for the line reading <tt>\usepackage[blind]{anonymize}</tt><br>
```

Comment:
can i suggest something to make all `tt` tags more visible?
I can edit the styles css to include:
```css
    background-color: whitesmoke;
    color: black;
    padding: 2px;
    border-radius: 4px;    
```

Reply:
nice idea! can you include it? Or should I?

Reply:
Just saw you committed thanks!